Migration.md

```diff
@@ -0,0 +1,325 @@
+    "name": "some_name"
+  },
+  "coverage": {
+    "format": 0
+  }
+}
+```
+
+Additionally, [`seismic`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.Seismic) objects also have coverage. They can be fetched from [`com.cognite.seismic.v1.SeismicService.SearchSeismics`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries).
+
+### Payload example
+#### v1:
+```
+{
+  "seismic_store": {
+    "name": "some_name"
+  },
+  "coverage": {
+    "format": 0
+  }
+}
+```
+
+## com.cognite.seismic.Query.GetFileLineRange
+Use [`com.cognite.seismic.v1.SeismicService.SearchSeismicStores`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries) with [`include_extent`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#searchseismicstoresrequest) set to `true`.
+
+### Payload example
+#### v0:
+```
+{
+  "file": {
+    "name": "some_name"
+  }
+}
+```
+#### v1:
+```
+{
+  "file": {
+    "name": "some_name"
+  },
+  "include_extent": true
+}
+```
+
+Additionally, [`seismic`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.Seismic) objects also have the line range. They can be fetched from [`com.cognite.seismic.v1.SeismicService.SearchSeismics`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries).
+
+### Payload example
+#### v1:
+```
+{
+  "seismic": {
+    "id": "1234"
+  },
+  "include_line_range": true
+}
+```
+
+## com.cognite.seismic.Query.GetCrosslinesByInline
+**Deprecated**
+
+See [`com.cognite.seismic.Query.GetFileLineRange`](##com.cognite.seismic.Query.GetFileLineRange)
+
+## com.cognite.seismic.Query.GetInlinesByCrossline
+**Deprecated**
+
+See [`com.cognite.seismic.Query.GetFileLineRange`](##com.cognite.seismic.Query.GetFileLineRange)
+
+## com.cognite.seismic.Query.GetTraceByCoordinates
+To get the closest trace to a given coordinate, [`com.cognite.seismic.v1.SeismicService.StreamTraces`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries) can be used specifying the [`geometry`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#streamtracesrequest) with [`interpolation_method`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.GeometryFilter) set to [`NEAREST_TRACE`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.InterpolationMethod), and a [`geometry`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.Geometry) containing the coordinate of interest.
+
+### Payload example
+#### v0:
+```
+{
+  "file": {
+    "name": "some_name",
+  },
+  "coordinates": {
+    "x": 10,
+    "y": 20
+  }
+}
+```
+#### v1:
+```
+{
+  "seismic": {
+    "id": "1234"
+  },
+  "geometry": {
+    "geometry": {
+      "wkt": {
+        "geometry": "POINT(10 20)"
+      }
+    }
+  }
+}
+```
+
+## com.cognite.seismic.Query.GetSliceByLine
+**Deprecated**
+
+See [`com.cognite.seismic.Query.GetVolume`](##com.cognite.seismic.Query.GetVolume)
```

Comment:
In these cases, I think it's better to directly reference `StreamTraces` over the double indirection through `GetVolume`.